src/System.Globalization/tests/NumberFormatInfo/NumberFormatInfoNumberGroupSizes.cs

```diff
@@ -67,21 +67,15 @@ public void NegTest3()
 
         // TestNumberGroupSizesLocale: Verify value of property NumberGroupSizes for specific locales
         [Theory]
-        [InlineData("en-US", 3, 0)]
-        [InlineData("ur-IN", 3, 2)]
-        [ActiveIssue(3243)]
-        public void TestNumberGroupSizesLocale(string locale, int primaryGroupSize, int secondaryGroupSize)
+        [InlineData("en-US")]
+        [InlineData("ur-IN")]
+        public void TestNumberGroupSizesLocale(string locale)
```

Comment:
It isn't possible to have differing InlineData between Windows8.1 and Windows10 (which is causing problems with the ur-IN culture).  And so we have to do runtime checks in the test code.  And once you do that, then [@tarekgh's comment](https://github.com/ellismg/corefx/pull/6#discussion-diff-39179342) comes in to play where we pull all the "differing data" into a centralized place instead of having "if (s_isWindows)" everywhere in the test code.